.github/workflows/SQL2022.yml

```diff
@@ -154,3 +181,28 @@ jobs:
         env:
           PASSWORD_AIRLINE_USER: "${{ env.dbPassword }}"
           PASSWORD_AIRLINE_USER_DBOWNER: "${{ env.dbPassword }}"
+
+      - name: Set up Python ${{ env.python-version }}
+        uses: actions/setup-python@v2
```

Comment:
nit: `actions/setup-python` is now on version 5. Can you check whether upgrading has any negative effects?